src/CommonITILObject.php

```diff
@@ -5210,6 +5210,13 @@ public function updateDateMod($ID, $no_stat_computation = false, $users_id_lastu
                 $update['users_id_lastupdater'] = $users_id_lastupdater;
             }
 
+            // set take into account delay stat
+            if ($no_stat_computation && $this->getType() == Ticket::class) {
+                if ($this->fields['takeintoaccount_delay_stat'] == 0) {
+                    $update['takeintoaccount_delay_stat'] = 0;
+                }
```

Comment:
As explained in the internal chat, this code does not make sense to me. If `$this->fields['takeintoaccount_delay_stat']` equals `0`, why should we update it to the same value ?

Also, `$no_stat_computation` probably means that we should not change the value of the `takeintoaccount_delay_stat` field, even if we set it to `0`.